src/utils/createLink.js

```diff
@@ -52,28 +53,36 @@ const createLinkCommunity = ({item, location, section}) => {
   });
 };
 
-const createLinkDocs = ({item, location, section}) => {
-  const isActive = isItemActive(location, item);
+const createLinkDocs = ({item, location, section, isActive}) => {
+  const active =
+    typeof isActive === 'boolean' ? isActive : isItemActive(location, item);
 
   return (
     <Link
-      css={[linkCss, isActive && activeLinkCss]}
+      css={[linkCss, active && activeLinkCss]}
       to={slugify(item.id, section.directory)}>
-      {isActive && <span css={activeLinkBefore} />}
+      {active && <span css={activeLinkBefore} />}
       {item.title}
     </Link>
   );
 };
 
-const createLinkTutorial = ({item, location, onLinkClick, section}) => {
-  const isActive = isItemActive(location, item);
+const createLinkTutorial = ({
+  item,
+  location,
+  onLinkClick,
+  section,
+  isActive,
+}) => {
+  const active =
+    typeof isActive === 'boolean' ? isActive : isItemActive(location, item);
```

Comment:
Would be nice if we didn't have to do this in all of the createLink* util methods.

What do you think about just making `isActive` a boolean parameter that's always expected, and:
* In the case of the scroll-sync decorator component, we could do a smarter check and pass along as a prop to `Section`.
* If no such prop is passed then `Section` can just call the default `isItemActive` util itself